### PR TITLE
Add OG meta tags

### PR DESCRIPTION
**Pull request by mrodessit**

### CODE REVIEW

File: src/components/PublicKeyTweet/index.tsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ const PublicKeyTweet: React.FC = ({
   message,
   publicKey,
 }) => {
-  const publicKeyUrl = `https://proofof.dog/addr?dogecoin=${publicKey}`
+  const publicKeyUrl = `https://proofof.dog/addr?dogname=${dogname}&dogecoin=${publicKey}`
```
Comment (allenday) [inline src/components/PublicKeyTweet/index.tsx]: reorder and ensure data are clean. 

- dogecoin=
- twitter= (lowercase, whitespace trimmed, @ symbol trimmed)
- dogname= (whitespace trimmed and truncated to max length 64)

File: src/containers/GenerateToken/UserInfo.tsx [hunk 2]
```diff
@@ -16,7 +16,7 @@ const UserInfo: React.FC = ({
 }) => {
   const pbKeyRef = useRef()
   const scKeyRef = useRef()
-  const publicKeyUrl = `https://proofof.dog/addr?dogecoin=${publicKey}`
+  const publicKeyUrl = `https://proofof.dog/addr?dogname=${dogname}&dogecoin=${publicKey}`
```
Comment (allenday) [inline src/containers/GenerateToken/UserInfo.tsx]: reorder and ensure data are clean. 

- dogecoin=
- twitter= (lowercase, whitespace trimmed, @ symbol trimmed)
- dogname= (whitespace trimmed and truncated to max length 64)

**Review by allenday — COMMENTED** [review]
can accept with some minor changes for debugging and sanitizing inputs

File: src/components/PublicKeyTweet/index.tsx (hunk 1 above)
Comment (allenday) [reply src/components/PublicKeyTweet/index.tsx]: trim twitter to max length 64 also

File: src/containers/GenerateToken/UserInfo.tsx (hunk 2 above)
Comment (allenday) [reply src/containers/GenerateToken/UserInfo.tsx]: trim twitter to max length 64 also

**Comment by mrodessit** [comment]
changes ready.

**Comment by mrodessit** [comment]
Make photo from camera feature & /getting-started styles fix

**Pull request merged**